### PR TITLE
Onboard servers with Configuration Manager using a task sequence

### PR DESCRIPTION
**Pull request by sebassem**

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+title: "Windows server with Configuration Manager using a Task Sequence"
+linkTitle: "Windows server with Configuration Manager using a Task Sequence"
+weight: 5
+description: >
+---
+## Connect an existing Windows server to Azure Arc using Configuration Manager with a Task Sequence
+
+The following README will guide you on how to connect a Windows machine to Azure Arc with a Task Sequence using Configuration Manager.
+
+This guide assumes that you already have an installation of [Microsoft Configuration Manager](https://docs.microsoft.com/en-us/mem/configmgr/core/understand/introduction), at least one active Windows server client, an active distribution point and a basic understanding of the product.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md:12]:
```suggestion
This guide assumes that you already have an installation of [Microsoft Configuration Manager](https://docs.microsoft.com/en-us/mem/configmgr/core/understand/introduction) and a basic understanding of the product, at least one active Windows server client, an active distribution point.
```

File: docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  ```shell
+  az provider register --namespace Microsoft.HybridCompute
+  ```
+
+  You can monitor the registration process with the following commands:
+
+    ```shell
+    az provider show -n Microsoft.HybridCompute -o table
+    ```
+
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md:33]:
```suggestion
You can monitor the registration process with the following commands:

```shell
az provider show -n Microsoft.HybridCompute -o table
```

```

File: docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+- Add any relevant information about this application. Keep the defaults for the installation program and install behavior and click “Next” to finalize the application.
+
+    ![Screenshot showing the application configuration](./05.png)
+
+    ![Screenshot showing the created application](./06.png)
+
+- Select the newly created application and click on “Distribute Content”.
+
+    ![Screenshot showing initiating a content distribution](./07.png)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md:95]: make the numbers bigger and more obvious, it is hard to catch

File: docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+- Click “Add”, select “Distribution Point” and select one or more distribution points where you would like to distribute the application to.
+
+    ![Screenshot showing the distribution point option](./10.png)
+
+    ![Screenshot showing the distribution point selection](./11.png)
+
+- Click “Next” to finalize the wizard and initate the content distributtion.
+
+    ![Screenshot showing a successful content distribution](./12.png)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md:113]:
```suggestion
- Keep the defaults and click “Next”.

    ![Screenshot showing the initial content distribution wizard](./08.png)

- Make sure to have the Azure Connected Machine agent application selected click “Next”.

    ![Screenshot showing the application being distributed](./09.png)

- Click “Add”, select “Distribution Point” and select one or more distribution points where you would like to distribute the application.

    ![Screenshot showing the distribution point option](./10.png)

    ![Screenshot showing the distribution point selection](./11.png)

- Click “Next” to finalize the wizard and initiate the content distribution.

    ![Screenshot showing a successful content distribution](./12.png)
```

File: docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+
+- After logging in, go to the “Software Library” workspace. Under “Application Management”, click  “Applications” and click "Create Application".
+
+    ![Screenshot showing creation of a new application](./03.png)
+
+- Browse to the location of the downloaded Azure Connected Machine agent msi package and click “Next”.
+
+    ![Screenshot showing the path of the agent](./04.png)
+
+- Add any relevant information about this application. Keep the defaults for the installation program and install behavior and click “Next” to finalize the application.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md:87]:
```suggestion
- Download the Azure Connected Machine agent package for Windows from the [Microsoft Download Center](https://aka.ms/AzureConnectedMachineAgent) and copy it to the Configuration Manager server.

- Login to the Configuration Manager console.

- After logging in, go to the “Software Library” workspace. Under “Application Management”, click  “Applications” and click "Create Application".

    ![Screenshot showing the creation of a new application](./03.png)

- Browse to the location of the downloaded Azure Connected Machine agent MSI package and click “Next”.

    ![Screenshot showing the path of the agent](./04.png)

- Add any relevant information about this application. Keep the defaults for the installation program and install behavior and click “Next” to finalize the application.
```

File: docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+
+    ![Screenshot showing selecting the enforcement method](./25.png)
+
+- Keep the defaults and finalize the Task Sequence deployment wizard.
+
+- Connect to the server to be onboarded and open “Software Center”. After the server's machine policy has been refreshed, you should see the Task Sequence deployment in the Applications available in Software Center.
+
+    ![Screenshot showing software center](./26.png)
+
+- Select on the Task Sequence deployment and click “Install”.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_servers/scaled_deployment/configuration_manager_scaled_tasksequence/_index.md:177]:
```suggestion
- Click on the Task Sequence deployment and click “Install”.
```

**Review by likamrat — CHANGES_REQUESTED** [review]
Great work, minor tweaks

**Comment by sebassem** [comment]
@likamrat all comments have been applied, thanks Lior

**Review by likamrat — APPROVED** [review]
LGTM

**Pull request merged**